src/index.html

```diff
@@ -7,6 +7,435 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__link"
+      >
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="moyo_logo"
+            >
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav_list">
+          <li class="header__nav_item">
+            <a href="#apple" class="header__nav_link is-active">Apple</a>
+          </li>
```

Comment:
you can use navigation as separated BEM block
```suggestion
      <nav class="nav">
        <ul class="nav_list">
          <li class="nav_item">
            <a href="#apple" class="nav_link is-active">Apple</a>
          </li>
```